rpcs3/Emu/RSX/VK/VKFormats.cpp

```diff
@@ -142,24 +141,24 @@ VkComponentMapping get_component_mapping(u32 format, u8 swizzle_mask)
 
 	case CELL_GCM_TEXTURE_W16_Z16_Y16_X16_FLOAT:
 		return { VK_COMPONENT_SWIZZLE_A, VK_COMPONENT_SWIZZLE_B, VK_COMPONENT_SWIZZLE_G, VK_COMPONENT_SWIZZLE_R };
-
+		
+	case CELL_GCM_TEXTURE_X32_FLOAT:
+		return { VK_COMPONENT_SWIZZLE_ONE, VK_COMPONENT_SWIZZLE_ONE, VK_COMPONENT_SWIZZLE_ONE, VK_COMPONENT_SWIZZLE_R };
```

Comment:
I reference the GL one which shown below 

case CELL_GCM_TEXTURE_X32_FLOAT: return { GL_RED, GL_ONE, GL_ONE, GL_ONE };